front/app/containers/Admin/formBuilder/components/FormBuilderSettings/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+// Components
+import ConfigMultiselectWithLocaleSwitcher from './ConfigMultiselectWithLocaleSwitcher';
+import LinearScaleSettings from './LinearScaleSettings';
+
+// Function to return additional settings based on input type
+export function getAdditionalSettings(type: any, locales: any, index: any) {
```

Comment:
We could probably use the `ICustomFieldInputType` here for the typings and add the others as-well.